rkt/images.go

```diff
@@ -636,6 +713,8 @@ func (f *fetcher) downloadHTTP(url, label string, out writeSyncer, etag string)
 		return nil, fmt.Errorf("bad HTTP status code: %d", res.StatusCode)
```

Comment:
If the error is `http.StatusRequestedRangeNotSatisfiable` (416), then there is a bug in the cache management (in rkt or in the server). Should we delete the partially downloaded file and the etag file in order to recover from that?